components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/configuration/PulsarConfiguration.java

```diff
@@ -354,4 +358,28 @@ public void setMessageRouter(MessageRouter messageRouter) {
         this.messageRouter = messageRouter;
     }
 
+    public Integer getMaxRedeliverCount() {
+        return maxRedeliverCount;
+    }
+
+    /**
```

Comment:
This javadoc and the option description are not the same, eg if the other options dont have javadoc then remove it also here, eg they are documented in the json/adoc file which is the important place